src/pages/AddItemPage/components/TagList.tsx

```diff
@@ -1,21 +1,21 @@
-import { useState } from "react";
+import React, { ChangeEvent, KeyboardEvent, useState } from "react";
 
-function TagList() {
-  const [tags, setTags] = useState("");
-  const [tagList, setTagList] = useState([]);
+const TagList: React.FC = () => {
+  const [tags, setTags] = useState<string>("");
```

Comment:
```suggestion
  const [tags, setTags] = useState("");
```
useState에 초기값을 넣어주면 자동으로 그 값으로 타입이 지정돼요!
그래서 useState에 여러 타입이 들어가는것이 아니면 생략도 가능하답니다!

Reply:
```
const initialItem: Item = {
  id: 0,
  name: "",
  description: "",
  price: 0,
  tags: [],
  images: [],
  favoriteCount: 0,
};

const ItemDetailPage = () => {
  const { productId } = useParams<{ productId: string }>();
  const [item, setItem] = useState<Item>(initialItem);
```

이런 경우에도 ` const [item, setItem] = useState(initialItem);` 이렇게 생략 가능한게 맞나요??

Reply:
네네 맞습니다!
<img width="561" alt="image" src="https://github.com/codeit-bootcamp-frontend/7-Sprint-Mission/assets/79738187/1c841954-995e-4441-8cdb-2c6c46210371">
item이나 setItem에 마우스 올려놓으면 타입을 보여주는데 Item type이 잘 들어가있는것을 볼 수 있으실거에요!